### DISCUSSION TITLE
where to set "temporary state"?

### DISCUSSION THREAD

**Discussion by starascendin** [illacloud/illa-builder#2144] [General]
Hi all,

i've been looking to set temporary state but i cannot find where it is or how to use it. Can anyone point me to the right loc?

**Comment by wenzhuo2022** [comment]
Hello, our Global data may meet your needs, add it through the location below. 
The screenshot: https://cloud-api.illacloud.com/drive/f/bd8ea186-5a6d-49f7-92b4-c4c1bfb39d88
